mkdocs.yml

```diff
@@ -41,6 +41,7 @@ pages:
   - User Guides:
     - gominer:
       - Pool-Mining:
+		- 'Linux Guide': 'mining/proof-of-work/pool-mining/gominer/linux.md'
```

Comment:
This really should be indented the same as the windows line right below it (8 spaces I think).